artserver/rockcraft.yaml

```
@@ -0,0 +1,82 @@
+# Based on https://github.com/kserve/kserve/blob/v0.13.0/python/pmml.Dockerfile
+#
+# See ../CONTRIBUTING.md for more details about the patterns used in this rock.
+# This rock is implemented with some atypical patterns due to the native of the upstream
+# Dockerfile.
+name: artserver
+summary: Art server for Kserve deployments
+description: "Kserve Art server"
+version: "0.13.0"
+license: Apache-2.0
+base: ubuntu@22.04
+run-user: _daemon_
+
+platforms:
+    amd64:
+
+
+services:
+  artserver:
```

Match the line breaks that we use on the other rocks
```suggestion
run-user: _daemon_
platforms:
    amd64:
services:
  artserver:
```